docs/organizer-guide/confs/cfp.rst

```diff
@@ -48,74 +52,85 @@ We use a tool called Pretalx, which lets us store, review, and record our select
 Each member of the review committee will need:
 
 * Access to `Pretalx <https://pretalx.com>`__
-* 6+ hours of time for proposal review
-* Optional review meeting (usually two hours)
+* Around 6 hours of time for proposal review
+* (Optional) Review meeting (usually two hours)
 
 Reviewing starts as soon as there are propsals to review, although most proposals usually come in in the last week or so.
+We usually end the Call for Proposals around 8 weeks before the conference, aim to hold the review meeting within one week, and give the speakers around a week to confirm, then reach out to alternates if needed.
 
 We encourage our reviewers to comment on each proposal as well as rate them, since it makes for a much more productive review meeting.
 
 Reviewing Guidelines
 ~~~~~~~~~~~~~~~~~~~~~
 
-**How we rate talks.** Please use the following rating system:
-
- 1. I will argue strongly against including this in the program.
- 2. I'm not a fan, but it wouldn't be the end of the world if we included this.
- 3. (DO NOT USE. Three means you don't have an opinion. We don't believe you. No threes.)
- 4. This is a decent talk. I'd be cool with including it.
- 5. I will fight to make sure this talk is included in the program.
-
-**Revisit your first ~15 talks.** When you're reviewing this many talks, it can take a while to get the feel of the rating system since, at first, you only have a small sample to compare to. To solve this problem, we recommend that, when you get to the end of your reviews, you go back and take another quick look at the first 15 or so talks you looked at, to make sure your rating holds up in the context of the whole pool.
-
-**The review meeting.** This can take up to two hours. We sort talks by **Median** rating, and move quite quickly through the top six or so talks (although if there is a **1** as well as a high score it might need some discussion).
-
-Dealing with conflicts of interest::
+Grade the quality of each proposal on a 5 to 1 grading scale for overall content, originality, relevance, and speaker(s):
+
+1. I will argue strongly against including this in the program.
+2. I'm not a fan, but it wouldn't be the end of the world if we included this.
+3. (DO NOT USE. Three means you don't have an opinion. We don't believe you. No threes.)
+4. This is a decent talk. I'd be cool with including it.
+5. I will fight to make sure this talk is included in the program.
+
+Reminders:
+
+* Leave comments for each proposal you review, with the reasons for your score.
+* Do not leave a score of 3.
+
+Review Process Best Practices
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Time Commitment**
+   Please plan on committing 4-6 hours total to review all of the submissions in your track, depending on the amount you have been assigned. Aim to do 10-15 sessions at a time then take a break. This helps prevent burnout and allows you to see more proposals with fresh eyes.
```

Comment:
By "sessions" do you mean maybe "10-15 minutes a session at a time" instead? Or do you mean "10-15 proposals at a time" then take a break?

Reply:
Also, for fun you could mention that this suggestion is based on the Pomodoro Technique and link out to the Wikipedia article about it: https://en.wikipedia.org/wiki/Pomodoro_Technique

(This is just a suggestion and I'll be totally fine if you disregard that suggestion, but I decided to pull out my Pomodoro timer when I read this because it made me think of it!)